app/models/user.rb

```diff
@@ -128,6 +130,20 @@ def self.create_or_promote_to_administrateur(email, password)
     user
   end
 
+  def self.create_or_promote_to_expert(email, password)
+    user = User
+      .create_with(password: password, confirmed_at: Time.zone.now)
+      .find_or_create_by(email: email)
+
+    if user.valid?
+      if user.expert_id.nil?
+        user.create_expert
```

Comment:
```suggestion
        user.create_expert!
```

Pour être homogene avec les autres méthodes et lever une exception si on a un soucis